tests/datasource/fluent/test_batch.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import annotations
```

Comment:
Thoughts on adding this file to type checking in `pyproject.toml`?

Reply:
The whole fluent directory is already under test.
```
files = [
    "great_expectations",
    "tests/datasource/fluent",
    ...
]
```